src/test/java/LogsApiClientTest.java

```diff
@@ -0,0 +1,103 @@
+import com.okta.sdk.clients.LogsApiClient;
+import com.okta.sdk.framework.ApiClientConfiguration;
+import com.okta.sdk.framework.FilterBuilder;
+import org.hamcrest.core.StringContains;
+import org.joda.time.DateTime;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * @author GuyZehavi
+ */
+public class LogsApiClientTest {
+
+    // ApiClientConfiguration
+    private static final String BASE_URL = "https://some-org.com";
```

Comment:
So when customer download and run mvn install, will the build fail because of the test?